ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockDataNode.java

```diff
@@ -35,36 +32,16 @@
 public class MockDataNode {
   private DataNodeId dataNodeId;
 
-  private Map<BlobId, MaterializedBlob> blobs;
```

Comment:
Reason is that, already BlobData has ByteBufferInputStream and hence we don't need an extra processing (since earlier we had to convert InputStream to ByteBufferInputStream). But this raises a question why BlobData has ByteBufferInputStream whereas BlobOutput had InputStream, for which I don't have an answer. Looks like you made the change for it (https://github.com/linkedin/ambry/pull/320/files). Do you know why this change was made. Sorry, I didn't realize.